styles.css

```diff
@@ -1,85 +1,262 @@
-@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:wght@200;300;400;500;600;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@200;300;400;500;600;700&family=Poppins:wght@200;300;600;800&family=Roboto:wght@700&display=swap');
 
 body { margin: 0%; }
 
 .container {
   display: flex;
   flex-direction: column;
-  border-radius: 2%;
   width: 375px;
   height: 100vh;
-  background-color: black;
+  background-color: #1c1a19;
+  background-image: url("illustration-header-mobile.svg");
+  background-repeat: no-repeat;
+  background-position: bottom right;
 }
-.icon { height: 70px; }
 
 .navigation {
   display: flex;
   justify-content: space-around;
   background-color: rgb(34, 33, 33);
 }
+.list { padding-left: 0%; }
 
-.menu {
-  height: 30px;
+.icon {
+  height: 20px;
+  align-self: center;
+  justify-items: center;
+  padding-right: 20%;
+}
+
+.center {
   display: flex;
+  justify-content: center;
+  padding-right: 10%;
+}
+
+li {
+  display: flex;
+  text-decoration: none;
+  justify-content: center;
+  padding-right: 10%;
+}
+
+.menu {
   align-self: center;
-  opacity: 0.6;
+  height: 30px;
+  opacity: 0.7;
 }
 
 .name {
-  margin-top: 25%;
-  color: orange;
-  font-size: 46px;
-  width: 70%;
-  height: 100px;
-  font-family: 'Crete Round', serif;
-  font-weight: lighter;
-  margin-bottom: 5%;
+  margin-top: 30%;
+  width: 85%;
+  justify-self: center;
+}
+
+.nameabout {
+  font-size: 72px;
+  line-height: 88px;
+  margin-top: 30%;
+  margin-bottom: 2%;
+  width: 90%;
+  align-self: center;
+}
+
+.sizetitle {
+  font-size: 56px;
+  line-height: 64px;
+}
+
+.stitlewh {
+  margin-top: 60%;
+  font-size: 42px;
+  line-height: 52px;
+  justify-content: center;
+  padding-bottom: 10%;
+}
+
+.stitlebl {
+  font-size: 32px;
+  line-height: 44px;
 }
 
-.title {
-  font-size: 20px;
-  color: white;
-  font-family: 'Poppins', sans-serif;
-  font-weight: 600;
+.fsizelist {
+  list-style: none;
+  font-size: 22px;
+  line-height: 42.67px;
+}
+
+.flisttitle {
+  list-style: none;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44.67px;
 }
 
 .profile {
-  font-family: 'Poppins', sans-serif;
-  color: white;
   display: flex;
   width: 90%;
-  height: 126px;
   font-size: 16px;
+  line-height: 24px;
   font-weight: 400;
+  padding-right: 10%;
+  margin-top: 2%;
 }
-
-.divlink {
-  display: flex;
```

Comment:
- [ ] Kindly make sure you stick as much as possible to the template provided to you in Figma.
Here's how your project looks like 
![Screenshot from 2022-09-25 00-27-58](https://user-images.githubusercontent.com/97100091/192120999-38aa89ce-1f38-4d8f-a6b2-1dc077f9c6b8.png)

Compared to the Figma design
![Screenshot from 2022-09-25 00-30-11](https://user-images.githubusercontent.com/97100091/192121010-2fdf135c-d9de-41f5-aa72-d27d6cf3a0ca.png)